pl-ai-chatbot-db-prod                | 
pl-ai-chatbot-db-prod                | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-chatbot-db-prod                | 
pl-ai-auth-db-prod                   | 
pl-ai-auth-db-prod                   | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-auth-db-prod                   | 
pl-ai-user-db-prod                   | 
pl-ai-user-db-prod                   | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-user-db-prod                   | 
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.539 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.544 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.544 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.553 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.549 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.549 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.549 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.550 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.550 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.550 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.559 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.565 UTC [29] LOG:  database system was shut down at 2025-07-15 10:40:32 UTC
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.559 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.570 UTC [29] LOG:  database system was shut down at 2025-07-15 10:40:24 UTC
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.571 UTC [28] LOG:  database system was shut down at 2025-07-15 10:40:25 UTC
pl-ai-auth-db-prod                   | 2025-07-15 10:44:43.584 UTC [1] LOG:  database system is ready to accept connections
pl-ai-user-db-prod                   | 2025-07-15 10:44:43.613 UTC [1] LOG:  database system is ready to accept connections
pl-ai-chatbot-db-prod                | 2025-07-15 10:44:43.615 UTC [1] LOG:  database system is ready to accept connections
pl-ai-rag-db-prod                    | 
pl-ai-rag-db-prod                    | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-rag-db-prod                    | 
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.379 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.379 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.379 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.392 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.399 UTC [28] LOG:  database system was shut down at 2025-07-15 10:40:33 UTC
pl-ai-rag-db-prod                    | 2025-07-15 10:44:44.411 UTC [1] LOG:  database system is ready to accept connections
pl-ai-classifier-db-prod             | 
pl-ai-classifier-db-prod             | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-classifier-db-prod             | 
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.356 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.356 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.356 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.361 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.367 UTC [27] LOG:  database system was shut down at 2025-07-15 10:42:53 UTC
pl-ai-classifier-db-prod             | 2025-07-15 10:44:45.404 UTC [1] LOG:  database system is ready to accept connections
pl-ai-analysis-db-prod               | 
pl-ai-analysis-db-prod               | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-analysis-db-prod               | 
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.697 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.697 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.697 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.702 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.712 UTC [29] LOG:  database system was shut down at 2025-07-15 10:40:33 UTC
pl-ai-analysis-db-prod               | 2025-07-15 10:44:45.733 UTC [1] LOG:  database system is ready to accept connections
pl-ai-image-generator-db-prod        | 
pl-ai-image-generator-db-prod        | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-image-generator-db-prod        | 
pl-ai-learning-db-prod               | 
pl-ai-learning-db-prod               | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-learning-db-prod               | 
pl-ai-resource-db-prod               | 
pl-ai-resource-db-prod               | PostgreSQL Database directory appears to contain a database; Skipping initialization
pl-ai-resource-db-prod               | 
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.031 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.043 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.043 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.046 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.052 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.052 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.052 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.055 UTC [28] LOG:  database system was shut down at 2025-07-15 10:40:33 UTC
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.067 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-image-generator-db-prod        | 2025-07-15 10:44:46.073 UTC [1] LOG:  database system is ready to accept connections
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.081 UTC [28] LOG:  database system was shut down at 2025-07-15 10:40:33 UTC
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.093 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.093 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
pl-ai-learning-db-prod               | 2025-07-15 10:44:46.098 UTC [1] LOG:  database system is ready to accept connections
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.103 UTC [1] LOG:  listening on IPv6 address "::", port 5432
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.106 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.114 UTC [29] LOG:  database system was shut down at 2025-07-15 10:42:53 UTC
pl-ai-resource-db-prod               | 2025-07-15 10:44:46.133 UTC [1] LOG:  database system is ready to accept connections
pl-ai-frontend-prod                  |  INFO  Accepting connections at http://localhost:3000
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.322148+00:00 [notice] <0.44.0> Application syslog exited with reason: stopped
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.329330+00:00 [notice] <0.230.0> Logging: switching to configured handler(s); following messages may not be visible in this log output
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.331062+00:00 [notice] <0.230.0> Logging: configured log handlers are now ACTIVE
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.639895+00:00 [info] <0.230.0> ra: starting system quorum_queues
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.640062+00:00 [info] <0.230.0> starting Ra system: quorum_queues in directory: /var/lib/rabbitmq/mnesia/rabbit@6c9dc50ec2e4/quorum/rabbit@6c9dc50ec2e4
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.740632+00:00 [info] <0.261.0> ra system 'quorum_queues' running pre init for 0 registered servers
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.761105+00:00 [info] <0.262.0> ra: meta data store initialised for system quorum_queues. 0 record(s) recovered
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.786422+00:00 [notice] <0.267.0> WAL: ra_log_wal init, open tbls: ra_log_open_mem_tables, closed tbls: ra_log_closed_mem_tables
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.793986+00:00 [info] <0.230.0> ra: starting system coordination
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.794054+00:00 [info] <0.230.0> starting Ra system: coordination in directory: /var/lib/rabbitmq/mnesia/rabbit@6c9dc50ec2e4/coordination/rabbit@6c9dc50ec2e4
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.795826+00:00 [info] <0.274.0> ra system 'coordination' running pre init for 0 registered servers
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.796803+00:00 [info] <0.275.0> ra: meta data store initialised for system coordination. 0 record(s) recovered
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.797209+00:00 [notice] <0.280.0> WAL: ra_coordination_log_wal init, open tbls: ra_coordination_log_open_mem_tables, closed tbls: ra_coordination_log_closed_mem_tables
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.800230+00:00 [info] <0.230.0> 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.800230+00:00 [info] <0.230.0>  Starting RabbitMQ 3.12.14 on Erlang 25.3.2.15 [jit]
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.800230+00:00 [info] <0.230.0>  Copyright (c) 2007-2024 Broadcom Inc and/or its subsidiaries
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.800230+00:00 [info] <0.230.0>  Licensed under the MPL 2.0. Website: https://rabbitmq.com
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.800321+00:00 [error] <0.230.0> This release series has reached end of life and is no longer supported. Please visit https://rabbitmq.com/versions.html to learn more and upgrade
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   ##  ##      RabbitMQ 3.12.14
pl-ai-rabbitmq-prod                  |   ##  ##
pl-ai-rabbitmq-prod                  |   ##########  Copyright (c) 2007-2024 Broadcom Inc and/or its subsidiaries
pl-ai-rabbitmq-prod                  |   ######  ##
pl-ai-rabbitmq-prod                  |   ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   Erlang:      25.3.2.15 [jit]
pl-ai-rabbitmq-prod                  |   TLS Library: OpenSSL - OpenSSL 3.1.7 3 Sep 2024
pl-ai-rabbitmq-prod                  |   Release series support status: out of support
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   Doc guides:  https://rabbitmq.com/documentation.html
pl-ai-rabbitmq-prod                  |   Support:     https://rabbitmq.com/contact.html
pl-ai-rabbitmq-prod                  |   Tutorials:   https://rabbitmq.com/getstarted.html
pl-ai-rabbitmq-prod                  |   Monitoring:  https://rabbitmq.com/monitoring.html
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   Logs: <stdout>
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   Config file(s): /etc/rabbitmq/conf.d/10-defaults.conf
pl-ai-rabbitmq-prod                  | 
pl-ai-rabbitmq-prod                  |   Starting broker...2025-07-15 10:44:51.816975+00:00 [info] <0.230.0> 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  node           : rabbit@6c9dc50ec2e4
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  home dir       : /var/lib/rabbitmq
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  config file(s) : /etc/rabbitmq/conf.d/10-defaults.conf
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  cookie hash    : lmgwLCdtKs1Gu48nXdZ5rQ==
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  log(s)         : <stdout>
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:51.816975+00:00 [info] <0.230.0>  data dir       : /var/lib/rabbitmq/mnesia/rabbit@6c9dc50ec2e4
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.042880+00:00 [info] <0.230.0> Running boot step pre_boot defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.043049+00:00 [info] <0.230.0> Running boot step rabbit_global_counters defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.044400+00:00 [info] <0.230.0> Running boot step rabbit_osiris_metrics defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.044567+00:00 [info] <0.230.0> Running boot step rabbit_core_metrics defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.045804+00:00 [info] <0.230.0> Running boot step rabbit_alarm defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.051617+00:00 [info] <0.299.0> Memory high watermark set to 1552 MiB (1627627520 bytes) of 3880 MiB (4069068800 bytes) total
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.055686+00:00 [info] <0.301.0> Enabling free disk space monitoring (disk free space: 20450652160, total memory: 4069068800)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.055754+00:00 [info] <0.301.0> Disk free limit set to 50MB
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.057819+00:00 [info] <0.230.0> Running boot step code_server_cache defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.057946+00:00 [info] <0.230.0> Running boot step file_handle_cache defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.058234+00:00 [info] <0.304.0> Limiting to approx 1048479 file handles (943629 sockets)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.058368+00:00 [info] <0.305.0> FHC read buffering: OFF
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.058413+00:00 [info] <0.305.0> FHC write buffering: ON
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.059435+00:00 [info] <0.230.0> Running boot step worker_pool defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.059531+00:00 [info] <0.282.0> Will use 2 processes for default worker pool
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.059584+00:00 [info] <0.282.0> Starting worker pool 'worker_pool' with 2 processes in it
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.060503+00:00 [info] <0.230.0> Running boot step database defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.061369+00:00 [info] <0.230.0> Node database directory at /var/lib/rabbitmq/mnesia/rabbit@6c9dc50ec2e4 is empty. Assuming we need to join an existing cluster or initialise from scratch...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.061442+00:00 [info] <0.230.0> Configured peer discovery backend: rabbit_peer_discovery_classic_config
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.061535+00:00 [info] <0.230.0> Will try to lock with peer discovery backend rabbit_peer_discovery_classic_config
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.061615+00:00 [info] <0.230.0> All discovered existing cluster peers:
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.061944+00:00 [info] <0.230.0> Discovered no peer nodes to cluster with. Some discovery backends can filter nodes out based on a readiness criteria. Enabling debug logging might help troubleshoot.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.064801+00:00 [notice] <0.44.0> Application mnesia exited with reason: stopped
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.151094+00:00 [info] <0.230.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.151249+00:00 [info] <0.230.0> Successfully synced tables from a peer
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.151905+00:00 [notice] <0.283.0> Feature flags: attempt to enable `stream_sac_coordinator_unblock_group`...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.194833+00:00 [notice] <0.283.0> Feature flags: `stream_sac_coordinator_unblock_group` enabled
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.195777+00:00 [notice] <0.283.0> Feature flags: attempt to enable `restart_streams`...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.238208+00:00 [notice] <0.283.0> Feature flags: `restart_streams` enabled
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.238670+00:00 [info] <0.230.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.239211+00:00 [info] <0.230.0> Successfully synced tables from a peer
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.244433+00:00 [info] <0.230.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.244660+00:00 [info] <0.230.0> Successfully synced tables from a peer
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.244756+00:00 [info] <0.230.0> Peer discovery backend rabbit_peer_discovery_classic_config does not support registration, skipping registration.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.244856+00:00 [info] <0.230.0> Will try to unlock with peer discovery backend rabbit_peer_discovery_classic_config
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.245719+00:00 [info] <0.230.0> Running boot step tracking_metadata_store defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.245986+00:00 [info] <0.480.0> Setting up a table for connection tracking on this node: tracked_connection
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.246057+00:00 [info] <0.480.0> Setting up a table for per-vhost connection counting on this node: tracked_connection_per_vhost
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.246248+00:00 [info] <0.480.0> Setting up a table for per-user connection counting on this node: tracked_connection_per_user
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.246505+00:00 [info] <0.480.0> Setting up a table for channel tracking on this node: tracked_channel
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.246720+00:00 [info] <0.480.0> Setting up a table for channel tracking on this node: tracked_channel_per_user
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.247557+00:00 [info] <0.230.0> Running boot step networking_metadata_store defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.247970+00:00 [info] <0.230.0> Running boot step feature_flags defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.248402+00:00 [info] <0.230.0> Running boot step codec_correctness_check defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.248455+00:00 [info] <0.230.0> Running boot step external_infrastructure defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.248674+00:00 [info] <0.230.0> Running boot step rabbit_event defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.249619+00:00 [info] <0.230.0> Running boot step rabbit_registry defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.250150+00:00 [info] <0.230.0> Running boot step rabbit_auth_mechanism_amqplain defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.250677+00:00 [info] <0.230.0> Running boot step rabbit_auth_mechanism_cr_demo defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.251043+00:00 [info] <0.230.0> Running boot step rabbit_auth_mechanism_plain defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.251644+00:00 [info] <0.230.0> Running boot step rabbit_exchange_type_direct defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.252190+00:00 [info] <0.230.0> Running boot step rabbit_exchange_type_fanout defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.252672+00:00 [info] <0.230.0> Running boot step rabbit_exchange_type_headers defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.253012+00:00 [info] <0.230.0> Running boot step rabbit_exchange_type_topic defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.253508+00:00 [info] <0.230.0> Running boot step rabbit_mirror_queue_mode_all defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.253847+00:00 [info] <0.230.0> Running boot step rabbit_mirror_queue_mode_exactly defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.254208+00:00 [info] <0.230.0> Running boot step rabbit_mirror_queue_mode_nodes defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.254551+00:00 [info] <0.230.0> Running boot step rabbit_priority_queue defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.254730+00:00 [info] <0.230.0> Priority queues enabled, real BQ is rabbit_variable_queue
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.255334+00:00 [info] <0.230.0> Running boot step rabbit_queue_location_client_local defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.255473+00:00 [info] <0.230.0> Running boot step rabbit_queue_location_min_masters defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.255573+00:00 [info] <0.230.0> Running boot step rabbit_queue_location_random defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.255708+00:00 [info] <0.230.0> Running boot step kernel_ready defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.255907+00:00 [info] <0.230.0> Running boot step rabbit_sysmon_minder defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.256504+00:00 [info] <0.230.0> Running boot step rabbit_epmd_monitor defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.257757+00:00 [info] <0.488.0> epmd monitor knows us, inter-node communication (distribution) port: 25672
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.257968+00:00 [info] <0.230.0> Running boot step guid_generator defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.260124+00:00 [info] <0.230.0> Running boot step rabbit_node_monitor defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.260701+00:00 [info] <0.492.0> Starting rabbit_node_monitor (in ignore mode)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.260921+00:00 [info] <0.230.0> Running boot step delegate_sup defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.261805+00:00 [info] <0.230.0> Running boot step rabbit_memory_monitor defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.262312+00:00 [info] <0.230.0> Running boot step rabbit_fifo_dlx_sup defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.262802+00:00 [info] <0.230.0> Running boot step core_initialized defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.262879+00:00 [info] <0.230.0> Running boot step rabbit_channel_tracking_handler defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.263071+00:00 [info] <0.230.0> Running boot step rabbit_connection_tracking_handler defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.263296+00:00 [info] <0.230.0> Running boot step rabbit_definitions_hashing defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.263604+00:00 [info] <0.230.0> Running boot step rabbit_exchange_parameters defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.264544+00:00 [info] <0.230.0> Running boot step rabbit_mirror_queue_misc defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.265136+00:00 [info] <0.230.0> Running boot step rabbit_policies defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.266058+00:00 [info] <0.230.0> Running boot step rabbit_policy defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.266183+00:00 [info] <0.230.0> Running boot step rabbit_queue_location_validator defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.266693+00:00 [info] <0.230.0> Running boot step rabbit_quorum_memory_manager defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.267090+00:00 [info] <0.230.0> Running boot step rabbit_stream_coordinator defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.267949+00:00 [info] <0.230.0> Running boot step rabbit_vhost_limit defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.268211+00:00 [info] <0.230.0> Running boot step rabbit_mgmt_reset_handler defined by app rabbitmq_management
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.268326+00:00 [info] <0.230.0> Running boot step rabbit_mgmt_db_handler defined by app rabbitmq_management_agent
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.268634+00:00 [info] <0.230.0> Management plugin: using rates mode 'basic'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.270005+00:00 [info] <0.230.0> Running boot step recovery defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.271665+00:00 [info] <0.230.0> Running boot step empty_db_check defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.271739+00:00 [info] <0.230.0> Will seed default virtual host and user...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.271974+00:00 [info] <0.230.0> Adding vhost '/' (description: 'Default virtual host', tags: [])
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.274992+00:00 [info] <0.230.0> Inserted a virtual host record {vhost,<<"/">>,[],
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.274992+00:00 [info] <0.230.0>                                       #{description =>
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.274992+00:00 [info] <0.230.0>                                             <<"Default virtual host">>,
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.274992+00:00 [info] <0.230.0>                                         tags => []}}
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.286022+00:00 [info] <0.537.0> Making sure data directory '/var/lib/rabbitmq/mnesia/rabbit@6c9dc50ec2e4/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L' for vhost '/' exists
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.287693+00:00 [info] <0.537.0> Setting segment_entry_count for vhost '/' with 0 queues to '2048'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.292767+00:00 [info] <0.537.0> Starting message stores for vhost '/'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.293610+00:00 [info] <0.547.0> Message store "628WB79CIFDYO9LJI6DKMI09L/msg_store_transient": using rabbit_msg_store_ets_index to provide index
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.295749+00:00 [info] <0.537.0> Started message store of type transient for vhost '/'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.296552+00:00 [info] <0.551.0> Message store "628WB79CIFDYO9LJI6DKMI09L/msg_store_persistent": using rabbit_msg_store_ets_index to provide index
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.297815+00:00 [warning] <0.551.0> Message store "628WB79CIFDYO9LJI6DKMI09L/msg_store_persistent": rebuilding indices from scratch
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.299383+00:00 [info] <0.537.0> Started message store of type persistent for vhost '/'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.300063+00:00 [info] <0.537.0> Recovering 0 queues of type rabbit_classic_queue took 11ms
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.300169+00:00 [info] <0.537.0> Recovering 0 queues of type rabbit_quorum_queue took 0ms
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.300326+00:00 [info] <0.537.0> Recovering 0 queues of type rabbit_stream_queue took 0ms
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.304472+00:00 [info] <0.230.0> Created user 'admin'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.306279+00:00 [info] <0.230.0> Successfully set user tags for user 'admin' to [administrator]
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.308373+00:00 [info] <0.230.0> Successfully set permissions for user 'admin' in virtual host '/' to '.*', '.*', '.*'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.308489+00:00 [info] <0.230.0> Running boot step rabbit_observer_cli defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.308697+00:00 [info] <0.230.0> Running boot step rabbit_looking_glass defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.308806+00:00 [info] <0.230.0> Running boot step rabbit_core_metrics_gc defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.309025+00:00 [info] <0.230.0> Running boot step background_gc defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.310386+00:00 [info] <0.230.0> Running boot step routing_ready defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.310484+00:00 [info] <0.230.0> Running boot step pre_flight defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.311167+00:00 [info] <0.230.0> Running boot step notify_cluster defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.311263+00:00 [info] <0.230.0> Running boot step networking defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.311551+00:00 [info] <0.230.0> Running boot step definition_import_worker_pool defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.314497+00:00 [info] <0.282.0> Starting worker pool 'definition_import_pool' with 2 processes in it
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.318221+00:00 [info] <0.230.0> Running boot step cluster_name defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.318531+00:00 [info] <0.230.0> Initialising internal cluster ID to 'rabbitmq-cluster-id-s_FI0vacVZsm3IWvIvEAHQ'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.320781+00:00 [info] <0.230.0> Running boot step direct_client defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.320924+00:00 [info] <0.230.0> Running boot step rabbit_maintenance_mode_state defined by app rabbit
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.321334+00:00 [info] <0.230.0> Creating table rabbit_node_maintenance_states for maintenance mode status
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.325991+00:00 [info] <0.230.0> Running boot step rabbit_management_load_definitions defined by app rabbitmq_management
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.326180+00:00 [info] <0.589.0> Resetting node maintenance status
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.348215+00:00 [info] <0.648.0> Management plugin: HTTP (non-TLS) listener started on port 15672
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.349063+00:00 [info] <0.676.0> Statistics database started.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.349241+00:00 [info] <0.675.0> Starting worker pool 'management_worker_pool' with 3 processes in it
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.365220+00:00 [info] <0.690.0> Prometheus metrics: HTTP (non-TLS) listener started on port 15692
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.365489+00:00 [info] <0.589.0> Ready to start client connection listeners
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.368479+00:00 [info] <0.734.0> started TCP listener on [::]:5672
pl-ai-rabbitmq-prod                  |  completed with 4 plugins.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.493690+00:00 [info] <0.589.0> Server startup complete; 4 plugins started.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.493690+00:00 [info] <0.589.0>  * rabbitmq_prometheus
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.493690+00:00 [info] <0.589.0>  * rabbitmq_management
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.493690+00:00 [info] <0.589.0>  * rabbitmq_management_agent
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.493690+00:00 [info] <0.589.0>  * rabbitmq_web_dispatch
pl-ai-rabbitmq-prod                  | 2025-07-15 10:44:56.527894+00:00 [info] <0.9.0> Time to start RabbitMQ: 9886553 us
pl-ai-auth-service-prod              | DATABASE_URL: 
pl-ai-auth-service-prod              | Waiting for database at :...
pl-ai-auth-service-prod              | Skipping database check for now...
pl-ai-auth-service-prod              | Database is up!
pl-ai-auth-service-prod              | Applying database migrations...
pl-ai-chatbot-service-prod           | Aspetto che il database sia pronto... (chatbot_db:5432)
pl-ai-chatbot-service-prod           | Database pronto!
pl-ai-image-generator-service-prod   | Waiting for Image Generator DB at image_generator_db:5432...
pl-ai-image-generator-service-prod   | Image Generator DB is up!
pl-ai-image-generator-service-prod   | Applying database migrations...
pl-ai-auth-service-prod              | Operations to perform:
pl-ai-auth-service-prod              |   Apply all migrations: admin, auth, contenttypes, sessions, token_blacklist, users_api
pl-ai-auth-service-prod              | Running migrations:
pl-ai-auth-service-prod              |   No migrations to apply.
pl-ai-auth-service-prod              | Starting Django development server...
pl-ai-image-generator-service-prod   | Operations to perform:
pl-ai-image-generator-service-prod   |   Apply all migrations: admin, auth, contenttypes, generator_api, sessions
pl-ai-image-generator-service-prod   | Running migrations:
pl-ai-image-generator-service-prod   |   No migrations to apply.
pl-ai-image-generator-service-prod   | Checking for secrets...
pl-ai-image-generator-service-prod   | Starting Django development server on port 8000...
pl-ai-chatbot-service-prod           | Operations to perform:
pl-ai-chatbot-service-prod           |   Apply all migrations: admin, auth, chatbot_api, contenttypes, sessions
pl-ai-chatbot-service-prod           | Running migrations:
pl-ai-chatbot-service-prod           |   No migrations to apply.
pl-ai-auth-service-prod              | Watching for file changes with StatReloader
pl-ai-auth-service-prod              | Performing system checks...
pl-ai-auth-service-prod              | 
pl-ai-auth-service-prod              | System check identified no issues (0 silenced).
pl-ai-auth-service-prod              | July 15, 2025 - 10:45:23
pl-ai-auth-service-prod              | Django version 4.2.23, using settings 'service_auth.settings'
pl-ai-auth-service-prod              | Starting development server at http://0.0.0.0:8000/
pl-ai-auth-service-prod              | Quit the server with CONTROL-C.
pl-ai-auth-service-prod              | 
pl-ai-image-generator-service-prod   | Watching for file changes with StatReloader
pl-ai-image-generator-service-prod   | Performing system checks...
pl-ai-image-generator-service-prod   | 
pl-ai-image-generator-service-prod   | System check identified no issues (0 silenced).
pl-ai-image-generator-service-prod   | July 15, 2025 - 10:45:24
pl-ai-image-generator-service-prod   | Django version 4.2.23, using settings 'service_config.settings'
pl-ai-image-generator-service-prod   | Starting development server at http://0.0.0.0:8000/
pl-ai-image-generator-service-prod   | Quit the server with CONTROL-C.
pl-ai-image-generator-service-prod   | 
pl-ai-chatbot-service-prod           | Watching for file changes with StatReloader
pl-ai-chatbot-service-prod           | Performing system checks...
pl-ai-chatbot-service-prod           | 
pl-ai-chatbot-service-prod           | System check identified no issues (0 silenced).
pl-ai-chatbot-service-prod           | July 15, 2025 - 10:45:27
pl-ai-chatbot-service-prod           | Django version 4.2.23, using settings 'chatbot_service.settings'
pl-ai-chatbot-service-prod           | Starting development server at http://0.0.0.0:8000/
pl-ai-chatbot-service-prod           | Quit the server with CONTROL-C.
pl-ai-chatbot-service-prod           | 
pl-ai-resource-manager-service-prod  | Resource Manager Service Entrypoint (Web Process)
pl-ai-resource-manager-service-prod  | Waiting for Resource DB at resource_db:5432...
pl-ai-resource-manager-service-prod  | Resource DB is up!
pl-ai-resource-manager-service-prod  | Applying database migrations...
pl-ai-resource-manager-service-prod  | Using separate DB variables: dbname=resource_db user=resource_user host=resource_db
pl-ai-classifier-service-prod        | Image Classifier Service Entrypoint
pl-ai-classifier-service-prod        | Waiting for Classifier DB at classifier_db:5432...
pl-ai-classifier-service-prod        | Classifier DB is up!
pl-ai-classifier-service-prod        | Waiting for Broker at rabbitmq:5672...
pl-ai-classifier-service-prod        | Broker is up!
pl-ai-classifier-service-prod        | Applying database migrations...
pl-ai-learning-service-prod          | Waiting for database...
pl-ai-rag-worker-prod                | Avviando Celery worker per RAG...
pl-ai-rag-worker-prod                | Aspettando RabbitMQ...
pl-ai-learning-service-prod          | Database connected!
pl-ai-learning-service-prod          | Running migrations...
pl-ai-rag-service-prod               | Avviando il servizio RAG web...
pl-ai-rag-service-prod               | Aspettando che il database sia pronto...
pl-ai-resource-manager-worker-prod   | Using separate DB variables: dbname=resource_db user=resource_user host=resource_db
pl-ai-user-service-prod              | User Service Entrypoint
pl-ai-user-service-prod              | Waiting for User DB at user_db:5432...
pl-ai-user-service-prod              | User DB is up!
pl-ai-user-service-prod              | Applying database migrations...
pl-ai-data-analysis-service-prod     | Data Analysis Service Entrypoint
pl-ai-data-analysis-service-prod     | Waiting for Analysis DB at analysis_db:5432...
pl-ai-data-analysis-service-prod     | Analysis DB is up!
pl-ai-data-analysis-service-prod     | Waiting for Broker at rabbitmq:5672...
pl-ai-data-analysis-service-prod     | Broker is up!
pl-ai-data-analysis-service-prod     | OpenAI secret found.
pl-ai-data-analysis-service-prod     | Applying database migrations...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:45:54.644679+00:00 [info] <0.753.0> accepting AMQP connection <0.753.0> (172.23.0.21:40178 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:45:54.654236+00:00 [info] <0.753.0> connection <0.753.0> (172.23.0.21:40178 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:45:54.686395+00:00 [warning] <0.753.0> closing AMQP connection <0.753.0> (172.23.0.21:40178 -> 172.23.0.5:5672, vhost: '/', user: 'admin'):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:45:54.686395+00:00 [warning] <0.753.0> client unexpectedly closed TCP connection
pl-ai-rag-worker-prod                | RabbitMQ pronto!
pl-ai-rag-worker-prod                | Verificando modello Sentence Transformers...
pl-ai-resource-manager-service-prod  | Operations to perform:
pl-ai-resource-manager-service-prod  |   Apply all migrations: admin, auth, contenttypes, resources_api, sessions
pl-ai-resource-manager-service-prod  | Running migrations:
pl-ai-resource-manager-service-prod  |   No migrations to apply.
pl-ai-resource-manager-service-prod  | Starting Django development server on port 8000...
pl-ai-learning-service-prod          | OpenAI client inizializzato con successo
pl-ai-learning-service-prod          | No changes detected
pl-ai-rag-service-prod               | Database pronto!
pl-ai-rag-service-prod               | Aspettando RabbitMQ...
pl-ai-resource-manager-worker-prod   | /usr/local/lib/python3.10/site-packages/celery/platforms.py:829: SecurityWarning: You're running the worker with superuser privileges: this is
pl-ai-resource-manager-worker-prod   | absolutely not recommended!
pl-ai-resource-manager-worker-prod   | 
pl-ai-resource-manager-worker-prod   | Please specify a different user using the --uid option.
pl-ai-resource-manager-worker-prod   | 
pl-ai-resource-manager-worker-prod   | User information: uid=0 euid=0 gid=0 egid=0
pl-ai-resource-manager-worker-prod   | 
pl-ai-resource-manager-worker-prod   |   warnings.warn(SecurityWarning(ROOT_DISCOURAGED.format(
pl-ai-resource-manager-worker-prod   |  
pl-ai-resource-manager-worker-prod   |  -------------- celery@5365f09f76f6 v5.3.6 (emerald-rush)
pl-ai-resource-manager-worker-prod   | --- ***** ----- 
pl-ai-resource-manager-worker-prod   | -- ******* ---- Linux-6.1.0-37-amd64-x86_64-with-glibc2.36 2025-07-15 10:46:03
pl-ai-resource-manager-worker-prod   | - *** --- * --- 
pl-ai-resource-manager-worker-prod   | - ** ---------- [config]
pl-ai-resource-manager-worker-prod   | - ** ---------- .> app:         service_config:0x7feacc9940d0
pl-ai-resource-manager-worker-prod   | - ** ---------- .> transport:   amqp://admin:**@rabbitmq:5672//
pl-ai-resource-manager-worker-prod   | - ** ---------- .> results:     disabled://
pl-ai-resource-manager-worker-prod   | - *** --- * --- .> concurrency: 2 (prefork)
pl-ai-resource-manager-worker-prod   | -- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
pl-ai-resource-manager-worker-prod   | --- ***** ----- 
pl-ai-resource-manager-worker-prod   |  -------------- [queues]
pl-ai-resource-manager-worker-prod   |                 .> resource_tasks   exchange=resource_tasks(direct) key=resource_tasks
pl-ai-resource-manager-worker-prod   |                 
pl-ai-resource-manager-worker-prod   | 
pl-ai-resource-manager-worker-prod   | [tasks]
pl-ai-resource-manager-worker-prod   |   . resources_api.tasks.process_uploaded_resource
pl-ai-resource-manager-worker-prod   |   . service_config.celery.debug_task
pl-ai-resource-manager-worker-prod   | 
pl-ai-user-service-prod              | Operations to perform:
pl-ai-user-service-prod              |   Apply all migrations: admin, auth, contenttypes, sessions, user_api
pl-ai-user-service-prod              | Running migrations:
pl-ai-user-service-prod              |   No migrations to apply.
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:04,757: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-resource-manager-worker-prod   | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-resource-manager-worker-prod   | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-resource-manager-worker-prod   | you should set broker_connection_retry_on_startup to True.
pl-ai-resource-manager-worker-prod   |   warnings.warn(
pl-ai-resource-manager-worker-prod   | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.852116+00:00 [info] <0.763.0> accepting AMQP connection <0.763.0> (172.23.0.17:45458 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.860461+00:00 [info] <0.766.0> accepting AMQP connection <0.766.0> (172.23.0.20:41128 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.862851+00:00 [error] <0.766.0> Error on AMQP connection <0.766.0> (172.23.0.20:41128 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.862851+00:00 [error] <0.766.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.864222+00:00 [info] <0.766.0> closing AMQP connection <0.766.0> (172.23.0.20:41128 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.880481+00:00 [info] <0.763.0> connection <0.763.0> (172.23.0.17:45458 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:04,884: INFO/MainProcess] Connected to amqp://admin:**@rabbitmq:5672//
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:04,887: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-resource-manager-worker-prod   | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-resource-manager-worker-prod   | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-resource-manager-worker-prod   | you should set broker_connection_retry_on_startup to True.
pl-ai-resource-manager-worker-prod   |   warnings.warn(
pl-ai-resource-manager-worker-prod   | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.913454+00:00 [info] <0.776.0> accepting AMQP connection <0.776.0> (172.23.0.17:45460 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:04.924941+00:00 [info] <0.776.0> connection <0.776.0> (172.23.0.17:45460 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:04,928: INFO/MainProcess] mingle: searching for neighbors
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:05.096130+00:00 [info] <0.794.0> accepting AMQP connection <0.794.0> (172.23.0.17:45466 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:05.104313+00:00 [info] <0.794.0> connection <0.794.0> (172.23.0.17:45466 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-user-service-prod              | Starting Django development server on port 8000...
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:06,136: INFO/MainProcess] mingle: all alone
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:06,216: INFO/MainProcess] celery@5365f09f76f6 ready.
pl-ai-resource-manager-service-prod  | Using separate DB variables: dbname=resource_db user=resource_user host=resource_db
pl-ai-classifier-service-prod        | 2025-07-15 10:46:08.478741: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
pl-ai-classifier-service-prod        | 2025-07-15 10:46:08.478815: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
pl-ai-classifier-service-prod        | 2025-07-15 10:46:08.529244: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:09.513417+00:00 [info] <0.844.0> accepting AMQP connection <0.844.0> (172.23.0.20:41142 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:09.518447+00:00 [error] <0.844.0> Error on AMQP connection <0.844.0> (172.23.0.20:41142 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:09.518447+00:00 [error] <0.844.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:09.519832+00:00 [info] <0.844.0> closing AMQP connection <0.844.0> (172.23.0.20:41142 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-learning-service-prod          | OpenAI client inizializzato con successo
pl-ai-learning-service-prod          | Operations to perform:
pl-ai-learning-service-prod          |   Apply all migrations: admin, auth, contenttypes, learning_api, sessions
pl-ai-learning-service-prod          | Running migrations:
pl-ai-learning-service-prod          |   No migrations to apply.
pl-ai-image-classifier-worker-prod   | 2025-07-15 10:46:11.473242: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
pl-ai-image-classifier-worker-prod   | 2025-07-15 10:46:11.474276: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
pl-ai-image-classifier-worker-prod   | 2025-07-15 10:46:11.495155: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
pl-ai-learning-service-prod          | Creating superuser...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:13.968652+00:00 [info] <0.849.0> accepting AMQP connection <0.849.0> (172.23.0.20:41148 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:13.976805+00:00 [error] <0.849.0> Error on AMQP connection <0.849.0> (172.23.0.20:41148 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:13.976805+00:00 [error] <0.849.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:13.979770+00:00 [info] <0.849.0> closing AMQP connection <0.849.0> (172.23.0.20:41148 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-resource-manager-service-prod  | Using separate DB variables: dbname=resource_db user=resource_user host=resource_db
pl-ai-learning-service-prod          | Superuser already exists
pl-ai-chatbot-service-prod           | [15/Jul/2025 10:46:15] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-auth-service-prod              | [15/Jul/2025 10:46:16] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-resource-manager-service-prod  | Watching for file changes with StatReloader
pl-ai-resource-manager-service-prod  | Performing system checks...
pl-ai-resource-manager-service-prod  | 
pl-ai-image-generator-service-prod   | [15/Jul/2025 10:46:18] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-data-analysis-worker-prod      | OpenAI client initialized in tasks.py.
pl-ai-data-analysis-service-prod     | OpenAI client initialized in tasks.py.
pl-ai-data-analysis-worker-prod      | OpenAI client initialized in views.
pl-ai-data-analysis-worker-prod      | /usr/local/lib/python3.10/site-packages/celery/platforms.py:829: SecurityWarning: You're running the worker with superuser privileges: this is
pl-ai-data-analysis-worker-prod      | absolutely not recommended!
pl-ai-data-analysis-worker-prod      | 
pl-ai-data-analysis-worker-prod      | Please specify a different user using the --uid option.
pl-ai-data-analysis-worker-prod      | 
pl-ai-data-analysis-worker-prod      | User information: uid=0 euid=0 gid=0 egid=0
pl-ai-data-analysis-worker-prod      | 
pl-ai-data-analysis-worker-prod      |   warnings.warn(SecurityWarning(ROOT_DISCOURAGED.format(
pl-ai-data-analysis-service-prod     | OpenAI client initialized in views.
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:19.066153+00:00 [info] <0.854.0> accepting AMQP connection <0.854.0> (172.23.0.20:52324 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:19.084489+00:00 [error] <0.854.0> Error on AMQP connection <0.854.0> (172.23.0.20:52324 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:19.084489+00:00 [error] <0.854.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:19.093402+00:00 [info] <0.854.0> closing AMQP connection <0.854.0> (172.23.0.20:52324 -> 172.23.0.5:5672)
pl-ai-learning-service-prod          | 
pl-ai-learning-service-prod          | 160 static files copied to '/app/staticfiles'.
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-data-analysis-worker-prod      |  
pl-ai-data-analysis-worker-prod      |  -------------- celery@1760242ebd54 v5.3.6 (emerald-rush)
pl-ai-data-analysis-worker-prod      | --- ***** ----- 
pl-ai-data-analysis-worker-prod      | -- ******* ---- Linux-6.1.0-37-amd64-x86_64-with-glibc2.36 2025-07-15 10:46:19
pl-ai-data-analysis-worker-prod      | - *** --- * --- 
pl-ai-data-analysis-worker-prod      | - ** ---------- [config]
pl-ai-data-analysis-worker-prod      | - ** ---------- .> app:         service_config:0x7f5cd9da8970
pl-ai-data-analysis-worker-prod      | - ** ---------- .> transport:   amqp://admin:**@rabbitmq:5672//
pl-ai-data-analysis-worker-prod      | - ** ---------- .> results:     disabled://
pl-ai-data-analysis-worker-prod      | - *** --- * --- .> concurrency: 1 (prefork)
pl-ai-data-analysis-worker-prod      | -- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
pl-ai-data-analysis-worker-prod      | --- ***** ----- 
pl-ai-data-analysis-worker-prod      |  -------------- [queues]
pl-ai-data-analysis-worker-prod      |                 .> analysis_tasks   exchange=analysis_tasks(direct) key=analysis_tasks
pl-ai-data-analysis-worker-prod      |                 
pl-ai-data-analysis-worker-prod      | 
pl-ai-data-analysis-worker-prod      | [tasks]
pl-ai-data-analysis-worker-prod      |   . analysis_api.tasks.generate_synthetic_csv_task
pl-ai-data-analysis-worker-prod      |   . analysis_api.tasks.run_analysis_task
pl-ai-data-analysis-worker-prod      | 
pl-ai-data-analysis-service-prod     | Operations to perform:
pl-ai-data-analysis-service-prod     |   Apply all migrations: admin, analysis_api, auth, contenttypes, sessions
pl-ai-data-analysis-service-prod     | Running migrations:
pl-ai-data-analysis-service-prod     |   No migrations to apply.
pl-ai-learning-service-prod          | Starting learning service...
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:20,056: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-data-analysis-worker-prod      | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-data-analysis-worker-prod      | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-data-analysis-worker-prod      | you should set broker_connection_retry_on_startup to True.
pl-ai-data-analysis-worker-prod      |   warnings.warn(
pl-ai-data-analysis-worker-prod      | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.082255+00:00 [info] <0.859.0> accepting AMQP connection <0.859.0> (172.23.0.24:55208 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.111086+00:00 [info] <0.859.0> connection <0.859.0> (172.23.0.24:55208 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:20,184: INFO/MainProcess] Connected to amqp://admin:**@rabbitmq:5672//
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:20,194: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-data-analysis-worker-prod      | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-data-analysis-worker-prod      | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-data-analysis-worker-prod      | you should set broker_connection_retry_on_startup to True.
pl-ai-data-analysis-worker-prod      |   warnings.warn(
pl-ai-data-analysis-worker-prod      | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.233547+00:00 [info] <0.867.0> accepting AMQP connection <0.867.0> (172.23.0.24:55210 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.236516+00:00 [info] <0.867.0> connection <0.867.0> (172.23.0.24:55210 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:20,237: INFO/MainProcess] mingle: searching for neighbors
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.294067+00:00 [info] <0.884.0> accepting AMQP connection <0.884.0> (172.23.0.24:55226 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:20.302387+00:00 [info] <0.884.0> connection <0.884.0> (172.23.0.24:55226 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:20,312: INFO/MainProcess] sync with celery@1760242ebd54
pl-ai-user-service-prod              | Watching for file changes with StatReloader
pl-ai-user-service-prod              | Performing system checks...
pl-ai-user-service-prod              | 
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:21,345: INFO/MainProcess] mingle: sync with 1 nodes
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:21,346: INFO/MainProcess] mingle: sync complete
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:21,407: INFO/MainProcess] celery@1760242ebd54 ready.
pl-ai-user-service-prod              | System check identified no issues (0 silenced).
pl-ai-data-analysis-service-prod     | Starting process type: web
pl-ai-data-analysis-service-prod     | Starting Django development server on port 8000...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:23.416795+00:00 [info] <0.932.0> accepting AMQP connection <0.932.0> (172.23.0.20:52332 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:23.419219+00:00 [error] <0.932.0> Error on AMQP connection <0.932.0> (172.23.0.20:52332 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:23.419219+00:00 [error] <0.932.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:23.423307+00:00 [info] <0.932.0> closing AMQP connection <0.932.0> (172.23.0.20:52332 -> 172.23.0.5:5672)
pl-ai-resource-manager-service-prod  | System check identified no issues (0 silenced).
pl-ai-user-service-prod              | July 15, 2025 - 10:46:23
pl-ai-user-service-prod              | Django version 4.2.23, using settings 'service_config.settings'
pl-ai-user-service-prod              | Starting development server at http://0.0.0.0:8000/
pl-ai-user-service-prod              | Quit the server with CONTROL-C.
pl-ai-user-service-prod              | 
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-learning-service-prod          | Watching for file changes with StatReloader
pl-ai-resource-manager-service-prod  | July 15, 2025 - 10:46:24
pl-ai-resource-manager-service-prod  | Django version 4.2.23, using settings 'service_config.settings'
pl-ai-resource-manager-service-prod  | Starting development server at http://0.0.0.0:8000/
pl-ai-resource-manager-service-prod  | Quit the server with CONTROL-C.
pl-ai-resource-manager-service-prod  | 
pl-ai-classifier-service-prod        | Operations to perform:
pl-ai-classifier-service-prod        |   Apply all migrations: admin, auth, classifier_api, contenttypes, sessions
pl-ai-classifier-service-prod        | Running migrations:
pl-ai-classifier-service-prod        |   No migrations to apply.
pl-ai-image-classifier-worker-prod   | /usr/local/lib/python3.10/site-packages/celery/platforms.py:829: SecurityWarning: You're running the worker with superuser privileges: this is
pl-ai-image-classifier-worker-prod   | absolutely not recommended!
pl-ai-image-classifier-worker-prod   | 
pl-ai-image-classifier-worker-prod   | Please specify a different user using the --uid option.
pl-ai-image-classifier-worker-prod   | 
pl-ai-image-classifier-worker-prod   | User information: uid=0 euid=0 gid=0 egid=0
pl-ai-image-classifier-worker-prod   | 
pl-ai-image-classifier-worker-prod   |   warnings.warn(SecurityWarning(ROOT_DISCOURAGED.format(
pl-ai-image-classifier-worker-prod   |  
pl-ai-image-classifier-worker-prod   |  -------------- celery@3994f31ae624 v5.3.6 (emerald-rush)
pl-ai-image-classifier-worker-prod   | --- ***** ----- 
pl-ai-image-classifier-worker-prod   | -- ******* ---- Linux-6.1.0-37-amd64-x86_64-with-glibc2.36 2025-07-15 10:46:26
pl-ai-image-classifier-worker-prod   | - *** --- * --- 
pl-ai-image-classifier-worker-prod   | - ** ---------- [config]
pl-ai-image-classifier-worker-prod   | - ** ---------- .> app:         service_config:0x7f3dd61cc970
pl-ai-image-classifier-worker-prod   | - ** ---------- .> transport:   amqp://admin:**@rabbitmq:5672//
pl-ai-image-classifier-worker-prod   | - ** ---------- .> results:     disabled://
pl-ai-image-classifier-worker-prod   | - *** --- * --- .> concurrency: 1 (prefork)
pl-ai-image-classifier-worker-prod   | -- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
pl-ai-image-classifier-worker-prod   | --- ***** ----- 
pl-ai-image-classifier-worker-prod   |  -------------- [queues]
pl-ai-image-classifier-worker-prod   |                 .> classifier_tasks exchange=classifier_tasks(direct) key=classifier_tasks
pl-ai-image-classifier-worker-prod   |                 
pl-ai-image-classifier-worker-prod   | 
pl-ai-image-classifier-worker-prod   | [tasks]
pl-ai-image-classifier-worker-prod   |   . classifier_api.tasks.train_classifier_task
pl-ai-image-classifier-worker-prod   |   . service_config.celery.debug_task
pl-ai-image-classifier-worker-prod   | 
pl-ai-rag-worker-prod                | Caricando modello: all-MiniLM-L6-v2
pl-ai-learning-service-prod          | OpenAI client inizializzato con successo
modules.json: 100%|██████████| 349/349 [00:00<00:00, 1.11MB/s]
config_sentence_transformers.json: 100%|██████████| 116/116 [00:00<00:00, 555kB/s]
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.364606+00:00 [info] <0.959.0> accepting AMQP connection <0.959.0> (172.23.0.20:45328 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.366994+00:00 [error] <0.959.0> Error on AMQP connection <0.959.0> (172.23.0.20:45328 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.366994+00:00 [error] <0.959.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.368622+00:00 [info] <0.959.0> closing AMQP connection <0.959.0> (172.23.0.20:45328 -> 172.23.0.5:5672)
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:27,370: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-image-classifier-worker-prod   | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-image-classifier-worker-prod   | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-image-classifier-worker-prod   | you should set broker_connection_retry_on_startup to True.
pl-ai-image-classifier-worker-prod   |   warnings.warn(
pl-ai-image-classifier-worker-prod   | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.390313+00:00 [info] <0.964.0> accepting AMQP connection <0.964.0> (172.23.0.22:34394 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.400726+00:00 [info] <0.964.0> connection <0.964.0> (172.23.0.22:34394 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:27,404: INFO/MainProcess] Connected to amqp://admin:**@rabbitmq:5672//
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:27,415: WARNING/MainProcess] /usr/local/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-image-classifier-worker-prod   | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-image-classifier-worker-prod   | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-image-classifier-worker-prod   | you should set broker_connection_retry_on_startup to True.
pl-ai-image-classifier-worker-prod   |   warnings.warn(
pl-ai-image-classifier-worker-prod   | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.444902+00:00 [info] <0.972.0> accepting AMQP connection <0.972.0> (172.23.0.22:34404 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.447907+00:00 [info] <0.972.0> connection <0.972.0> (172.23.0.22:34404 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:27,449: INFO/MainProcess] mingle: searching for neighbors
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
README.md: 10.5kB [00:00, 8.95MB/s]  | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.532992+00:00 [info] <0.989.0> accepting AMQP connection <0.989.0> (172.23.0.22:34420 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:27.544779+00:00 [info] <0.989.0> connection <0.989.0> (172.23.0.22:34420 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:27,555: INFO/MainProcess] sync with celery@3994f31ae624
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:27,555: INFO/MainProcess] sync with celery@3994f31ae624
sentence_bert_config.json: 100%|██████████| 53.0/53.0 [00:00<00:00, 233kB/s]
config.json: 100%|██████████| 612/612 [00:00<00:00, 1.17MB/s]
pl-ai-classifier-service-prod        | Starting process type: web
pl-ai-classifier-service-prod        | Starting Django development server on port 8000...
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:28,572: INFO/MainProcess] mingle: sync with 2 nodes
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:28,572: INFO/MainProcess] mingle: sync complete
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:28,613: INFO/MainProcess] celery@3994f31ae624 ready.
pl-ai-data-analysis-service-prod     | Watching for file changes with StatReloader
pl-ai-data-analysis-service-prod     | Performing system checks...
pl-ai-data-analysis-service-prod     | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:30.438155+00:00 [info] <0.1037.0> accepting AMQP connection <0.1037.0> (172.23.0.20:45342 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:30.441558+00:00 [error] <0.1037.0> Error on AMQP connection <0.1037.0> (172.23.0.20:45342 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:30.441558+00:00 [error] <0.1037.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:30.443407+00:00 [info] <0.1037.0> closing AMQP connection <0.1037.0> (172.23.0.20:45342 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
model.safetensors: 100%|██████████| 90.9M/90.9M [00:01<00:00, 48.1MB/s]
pl-ai-classifier-service-prod        | Watching for file changes with StatReloader
pl-ai-classifier-service-prod        | Performing system checks...
pl-ai-classifier-service-prod        | 
tokenizer_config.json: 100%|██████████| 350/350 [00:00<00:00, 1.99MB/s]
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:33.338007+00:00 [info] <0.1042.0> accepting AMQP connection <0.1042.0> (172.23.0.20:45344 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:33.340365+00:00 [error] <0.1042.0> Error on AMQP connection <0.1042.0> (172.23.0.20:45344 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:33.340365+00:00 [error] <0.1042.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:33.341849+00:00 [info] <0.1042.0> closing AMQP connection <0.1042.0> (172.23.0.20:45344 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
vocab.txt: 232kB [00:00, 6.79MB/s]   | 
tokenizer.json: 466kB [00:00, 962kB/s] 
special_tokens_map.json: 100%|██████████| 112/112 [00:00<00:00, 196kB/s]
config.json: 100%|██████████| 190/190 [00:00<00:00, 720kB/s]
pl-ai-rag-worker-prod                | Modello caricato con successo!
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:36.120518+00:00 [info] <0.1047.0> accepting AMQP connection <0.1047.0> (172.23.0.20:53754 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:36.123564+00:00 [error] <0.1047.0> Error on AMQP connection <0.1047.0> (172.23.0.20:53754 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:36.123564+00:00 [error] <0.1047.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:36.124914+00:00 [info] <0.1047.0> closing AMQP connection <0.1047.0> (172.23.0.20:53754 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-data-analysis-service-prod     | OpenAI client initialized in tasks.py.
pl-ai-classifier-service-prod        | 2025-07-15 10:46:36.512041: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
pl-ai-classifier-service-prod        | 2025-07-15 10:46:36.513119: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
pl-ai-data-analysis-service-prod     | OpenAI client initialized in views.
pl-ai-data-analysis-service-prod     | System check identified no issues (0 silenced).
pl-ai-classifier-service-prod        | 2025-07-15 10:46:36.577380: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
pl-ai-data-analysis-service-prod     | July 15, 2025 - 10:46:36
pl-ai-data-analysis-service-prod     | Django version 4.2.23, using settings 'service_config.settings'
pl-ai-data-analysis-service-prod     | Starting development server at http://0.0.0.0:8000/
pl-ai-data-analysis-service-prod     | Quit the server with CONTROL-C.
pl-ai-data-analysis-service-prod     | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:38.763727+00:00 [info] <0.1052.0> accepting AMQP connection <0.1052.0> (172.23.0.20:53770 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:38.767477+00:00 [error] <0.1052.0> Error on AMQP connection <0.1052.0> (172.23.0.20:53770 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:38.767477+00:00 [error] <0.1052.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:38.768689+00:00 [info] <0.1052.0> closing AMQP connection <0.1052.0> (172.23.0.20:53770 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-classifier-service-prod        | System check identified no issues (0 silenced).
pl-ai-classifier-service-prod        | July 15, 2025 - 10:46:40
pl-ai-classifier-service-prod        | Django version 4.2.23, using settings 'service_config.settings'
pl-ai-classifier-service-prod        | Starting development server at http://0.0.0.0:8000/
pl-ai-classifier-service-prod        | Quit the server with CONTROL-C.
pl-ai-classifier-service-prod        | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:41.535145+00:00 [info] <0.1057.0> accepting AMQP connection <0.1057.0> (172.23.0.20:53782 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:41.539105+00:00 [error] <0.1057.0> Error on AMQP connection <0.1057.0> (172.23.0.20:53782 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:41.539105+00:00 [error] <0.1057.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:41.540853+00:00 [info] <0.1057.0> closing AMQP connection <0.1057.0> (172.23.0.20:53782 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rag-worker-prod                | Docker Secret file non trovato: /run/secrets/OPENAI_API_KEY
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:44.130724+00:00 [info] <0.1062.0> accepting AMQP connection <0.1062.0> (172.23.0.20:53788 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:44.140473+00:00 [error] <0.1062.0> Error on AMQP connection <0.1062.0> (172.23.0.20:53788 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:44.140473+00:00 [error] <0.1062.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:44.141743+00:00 [info] <0.1062.0> closing AMQP connection <0.1062.0> (172.23.0.20:53788 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:47.041507+00:00 [info] <0.1068.0> accepting AMQP connection <0.1068.0> (172.23.0.20:39818 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:47.051938+00:00 [error] <0.1068.0> Error on AMQP connection <0.1068.0> (172.23.0.20:39818 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:47.051938+00:00 [error] <0.1068.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:47.052138+00:00 [info] <0.1068.0> closing AMQP connection <0.1068.0> (172.23.0.20:39818 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-resource-manager-service-prod  | [15/Jul/2025 10:46:48] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:49.669028+00:00 [info] <0.1077.0> accepting AMQP connection <0.1077.0> (172.23.0.20:39824 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:49.671430+00:00 [error] <0.1077.0> Error on AMQP connection <0.1077.0> (172.23.0.20:39824 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:49.671430+00:00 [error] <0.1077.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:49.672597+00:00 [info] <0.1077.0> closing AMQP connection <0.1077.0> (172.23.0.20:39824 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-classifier-service-prod        | [15/Jul/2025 10:46:51] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:52.643324+00:00 [info] <0.1082.0> accepting AMQP connection <0.1082.0> (172.23.0.20:39834 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:52.645392+00:00 [error] <0.1082.0> Error on AMQP connection <0.1082.0> (172.23.0.20:39834 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:52.645392+00:00 [error] <0.1082.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:52.647869+00:00 [info] <0.1082.0> closing AMQP connection <0.1082.0> (172.23.0.20:39834 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rag-worker-prod                | INFO 2025-07-15 12:46:52,867 loader 1 139729587346304 Loading faiss with AVX2 support.
pl-ai-rag-worker-prod                | INFO 2025-07-15 12:46:52,939 loader 1 139729587346304 Successfully loaded faiss with AVX2 support.
pl-ai-learning-service-prod          | "GET /admin/login/ HTTP/1.1" 200 4193
pl-ai-rag-worker-prod                | /usr/local/lib/python3.11/site-packages/celery/platforms.py:829: SecurityWarning: You're running the worker with superuser privileges: this is
pl-ai-rag-worker-prod                | absolutely not recommended!
pl-ai-rag-worker-prod                | 
pl-ai-rag-worker-prod                | Please specify a different user using the --uid option.
pl-ai-rag-worker-prod                | 
pl-ai-rag-worker-prod                | User information: uid=0 euid=0 gid=0 egid=0
pl-ai-rag-worker-prod                | 
pl-ai-rag-worker-prod                |   warnings.warn(SecurityWarning(ROOT_DISCOURAGED.format(
pl-ai-rag-worker-prod                |  
pl-ai-rag-worker-prod                |  -------------- celery@7413bd14fa4e v5.3.4 (emerald-rush)
pl-ai-rag-worker-prod                | --- ***** ----- 
pl-ai-rag-worker-prod                | -- ******* ---- Linux-6.1.0-37-amd64-x86_64-with-glibc2.36 2025-07-15 12:46:53
pl-ai-rag-worker-prod                | - *** --- * --- 
pl-ai-rag-worker-prod                | - ** ---------- [config]
pl-ai-rag-worker-prod                | - ** ---------- .> app:         rag_service:0x7f1552b75dd0
pl-ai-rag-worker-prod                | - ** ---------- .> transport:   amqp://admin:**@rabbitmq:5672//
pl-ai-rag-worker-prod                | - ** ---------- .> results:     rpc://
pl-ai-rag-worker-prod                | - *** --- * --- .> concurrency: 1 (prefork)
pl-ai-rag-worker-prod                | -- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
pl-ai-rag-worker-prod                | --- ***** ----- 
pl-ai-rag-worker-prod                |  -------------- [queues]
pl-ai-rag-worker-prod                |                 .> rag_tasks        exchange=rag_tasks(direct) key=rag_tasks
pl-ai-rag-worker-prod                |                 
pl-ai-rag-worker-prod                | 
pl-ai-rag-worker-prod                | [tasks]
pl-ai-rag-worker-prod                |   . rag_api.tasks.cleanup_failed_documents
pl-ai-rag-worker-prod                |   . rag_api.tasks.process_document
pl-ai-rag-worker-prod                |   . rag_api.tasks.process_query
pl-ai-rag-worker-prod                |   . rag_api.tasks.process_rag_document_task
pl-ai-rag-worker-prod                |   . rag_api.tasks.reprocess_document_task
pl-ai-rag-worker-prod                | 
pl-ai-rag-worker-prod                | [2025-07-15 12:46:54,121: WARNING/MainProcess] /usr/local/lib/python3.11/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-rag-worker-prod                | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-rag-worker-prod                | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-rag-worker-prod                | you should set broker_connection_retry_on_startup to True.
pl-ai-rag-worker-prod                |   warnings.warn(
pl-ai-rag-worker-prod                | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.134097+00:00 [info] <0.1087.0> accepting AMQP connection <0.1087.0> (172.23.0.21:39020 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.137862+00:00 [info] <0.1087.0> connection <0.1087.0> (172.23.0.21:39020 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-rag-worker-prod                | [2025-07-15 12:46:54,138: INFO/MainProcess] Connected to amqp://admin:**@rabbitmq:5672//
pl-ai-rag-worker-prod                | [2025-07-15 12:46:54,141: WARNING/MainProcess] /usr/local/lib/python3.11/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
pl-ai-rag-worker-prod                | whether broker connection retries are made during startup in Celery 6.0 and above.
pl-ai-rag-worker-prod                | If you wish to retain the existing behavior for retrying connections on startup,
pl-ai-rag-worker-prod                | you should set broker_connection_retry_on_startup to True.
pl-ai-rag-worker-prod                |   warnings.warn(
pl-ai-rag-worker-prod                | 
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.145347+00:00 [info] <0.1095.0> accepting AMQP connection <0.1095.0> (172.23.0.21:39024 -> 172.23.0.5:5672)
pl-ai-rag-worker-prod                | [2025-07-15 12:46:54,148: INFO/MainProcess] mingle: searching for neighbors
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.147763+00:00 [info] <0.1095.0> connection <0.1095.0> (172.23.0.21:39024 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.168936+00:00 [info] <0.1112.0> accepting AMQP connection <0.1112.0> (172.23.0.21:39030 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:54.176623+00:00 [info] <0.1112.0> connection <0.1112.0> (172.23.0.21:39030 -> 172.23.0.5:5672): user 'admin' authenticated and granted access to vhost '/'
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:54,180: INFO/MainProcess] sync with celery@7413bd14fa4e
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:54,182: INFO/MainProcess] sync with celery@7413bd14fa4e
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:54,182: INFO/MainProcess] sync with celery@7413bd14fa4e
pl-ai-user-service-prod              | [15/Jul/2025 10:46:54] "GET /admin/login/ HTTP/1.1" 200 4178
pl-ai-data-analysis-service-prod     | [15/Jul/2025 10:46:54] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:55.112418+00:00 [info] <0.1124.0> accepting AMQP connection <0.1124.0> (172.23.0.20:36514 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:55.115084+00:00 [error] <0.1124.0> Error on AMQP connection <0.1124.0> (172.23.0.20:36514 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:55.115084+00:00 [error] <0.1124.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:55.116569+00:00 [info] <0.1124.0> closing AMQP connection <0.1124.0> (172.23.0.20:36514 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rag-worker-prod                | [2025-07-15 12:46:55,209: INFO/MainProcess] mingle: sync with 3 nodes
pl-ai-rag-worker-prod                | [2025-07-15 12:46:55,210: INFO/MainProcess] mingle: sync complete
pl-ai-data-analysis-worker-prod      | [2025-07-15 10:46:55,234: WARNING/MainProcess] Substantial drift from celery@7413bd14fa4e may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 10:46:55.233881 recv: 2025-07-15 12:46:55.229697]
pl-ai-image-classifier-worker-prod   | [2025-07-15 10:46:55,232: WARNING/MainProcess] Substantial drift from celery@7413bd14fa4e may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 10:46:55.232783 recv: 2025-07-15 12:46:55.229697]
pl-ai-resource-manager-worker-prod   | [2025-07-15 10:46:55,235: WARNING/MainProcess] Substantial drift from celery@7413bd14fa4e may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 10:46:55.232165 recv: 2025-07-15 12:46:55.229697]
pl-ai-rag-worker-prod                | [2025-07-15 12:46:55,243: INFO/MainProcess] celery@7413bd14fa4e ready.
pl-ai-rag-worker-prod                | [2025-07-15 12:46:55,458: WARNING/MainProcess] Substantial drift from celery@1760242ebd54 may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 12:46:55.458170 recv: 2025-07-15 10:46:55.456124]
pl-ai-rag-worker-prod                | [2025-07-15 12:46:56,281: WARNING/MainProcess] Substantial drift from celery@5365f09f76f6 may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 12:46:56.281769 recv: 2025-07-15 10:46:56.279840]
pl-ai-rag-worker-prod                | [2025-07-15 12:46:56,633: WARNING/MainProcess] Substantial drift from celery@3994f31ae624 may mean clocks are out of sync.  Current drift is 7200 seconds.  [orig: 2025-07-15 12:46:56.633666 recv: 2025-07-15 10:46:56.630884]
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:57.573178+00:00 [info] <0.1169.0> accepting AMQP connection <0.1169.0> (172.23.0.20:36530 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:57.576075+00:00 [error] <0.1169.0> Error on AMQP connection <0.1169.0> (172.23.0.20:36530 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:57.576075+00:00 [error] <0.1169.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:46:57.577625+00:00 [info] <0.1169.0> closing AMQP connection <0.1169.0> (172.23.0.20:36530 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:00.057925+00:00 [info] <0.1174.0> accepting AMQP connection <0.1174.0> (172.23.0.20:36544 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:00.060515+00:00 [error] <0.1174.0> Error on AMQP connection <0.1174.0> (172.23.0.20:36544 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:00.060515+00:00 [error] <0.1174.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:00.061428+00:00 [info] <0.1174.0> closing AMQP connection <0.1174.0> (172.23.0.20:36544 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:02.447244+00:00 [info] <0.1179.0> accepting AMQP connection <0.1179.0> (172.23.0.20:36556 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:02.449661+00:00 [error] <0.1179.0> Error on AMQP connection <0.1179.0> (172.23.0.20:36556 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:02.449661+00:00 [error] <0.1179.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:02.450706+00:00 [info] <0.1179.0> closing AMQP connection <0.1179.0> (172.23.0.20:36556 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:04.965470+00:00 [info] <0.1184.0> accepting AMQP connection <0.1184.0> (172.23.0.20:51466 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:04.968031+00:00 [error] <0.1184.0> Error on AMQP connection <0.1184.0> (172.23.0.20:51466 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:04.968031+00:00 [error] <0.1184.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:04.969222+00:00 [info] <0.1184.0> closing AMQP connection <0.1184.0> (172.23.0.20:51466 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:07.402660+00:00 [info] <0.1189.0> accepting AMQP connection <0.1189.0> (172.23.0.20:51468 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:07.419908+00:00 [error] <0.1189.0> Error on AMQP connection <0.1189.0> (172.23.0.20:51468 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:07.419908+00:00 [error] <0.1189.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:07.420143+00:00 [info] <0.1189.0> closing AMQP connection <0.1189.0> (172.23.0.20:51468 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:09.814384+00:00 [info] <0.1194.0> accepting AMQP connection <0.1194.0> (172.23.0.20:51478 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:09.817124+00:00 [error] <0.1194.0> Error on AMQP connection <0.1194.0> (172.23.0.20:51478 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:09.817124+00:00 [error] <0.1194.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:09.818307+00:00 [info] <0.1194.0> closing AMQP connection <0.1194.0> (172.23.0.20:51478 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:12.246786+00:00 [info] <0.1199.0> accepting AMQP connection <0.1199.0> (172.23.0.20:51482 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:12.249002+00:00 [error] <0.1199.0> Error on AMQP connection <0.1199.0> (172.23.0.20:51482 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:12.249002+00:00 [error] <0.1199.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:12.250106+00:00 [info] <0.1199.0> closing AMQP connection <0.1199.0> (172.23.0.20:51482 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:14.800825+00:00 [info] <0.1204.0> accepting AMQP connection <0.1204.0> (172.23.0.20:35220 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:14.802995+00:00 [error] <0.1204.0> Error on AMQP connection <0.1204.0> (172.23.0.20:35220 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:14.802995+00:00 [error] <0.1204.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:14.804235+00:00 [info] <0.1204.0> closing AMQP connection <0.1204.0> (172.23.0.20:35220 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-chatbot-service-prod           | [15/Jul/2025 10:47:15] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-auth-service-prod              | [15/Jul/2025 10:47:16] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:17.200418+00:00 [info] <0.1209.0> accepting AMQP connection <0.1209.0> (172.23.0.20:35222 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:17.203009+00:00 [error] <0.1209.0> Error on AMQP connection <0.1209.0> (172.23.0.20:35222 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:17.203009+00:00 [error] <0.1209.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:17.204075+00:00 [info] <0.1209.0> closing AMQP connection <0.1209.0> (172.23.0.20:35222 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-image-generator-service-prod   | [15/Jul/2025 10:47:18] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:19.637012+00:00 [info] <0.1214.0> accepting AMQP connection <0.1214.0> (172.23.0.20:35230 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:19.639675+00:00 [error] <0.1214.0> Error on AMQP connection <0.1214.0> (172.23.0.20:35230 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:19.639675+00:00 [error] <0.1214.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:19.641085+00:00 [info] <0.1214.0> closing AMQP connection <0.1214.0> (172.23.0.20:35230 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:22.059089+00:00 [info] <0.1219.0> accepting AMQP connection <0.1219.0> (172.23.0.20:35234 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:22.061720+00:00 [error] <0.1219.0> Error on AMQP connection <0.1219.0> (172.23.0.20:35234 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:22.061720+00:00 [error] <0.1219.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:22.063015+00:00 [info] <0.1219.0> closing AMQP connection <0.1219.0> (172.23.0.20:35234 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:24.455484+00:00 [info] <0.1224.0> accepting AMQP connection <0.1224.0> (172.23.0.20:35244 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:24.465127+00:00 [error] <0.1224.0> Error on AMQP connection <0.1224.0> (172.23.0.20:35244 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:24.465127+00:00 [error] <0.1224.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:24.468075+00:00 [info] <0.1224.0> closing AMQP connection <0.1224.0> (172.23.0.20:35244 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:26.868687+00:00 [info] <0.1229.0> accepting AMQP connection <0.1229.0> (172.23.0.20:41504 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:26.871345+00:00 [error] <0.1229.0> Error on AMQP connection <0.1229.0> (172.23.0.20:41504 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:26.871345+00:00 [error] <0.1229.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:26.872613+00:00 [info] <0.1229.0> closing AMQP connection <0.1229.0> (172.23.0.20:41504 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:29.344935+00:00 [info] <0.1234.0> accepting AMQP connection <0.1234.0> (172.23.0.20:41508 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:29.348008+00:00 [error] <0.1234.0> Error on AMQP connection <0.1234.0> (172.23.0.20:41508 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:29.348008+00:00 [error] <0.1234.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:29.349192+00:00 [info] <0.1234.0> closing AMQP connection <0.1234.0> (172.23.0.20:41508 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:31.821165+00:00 [info] <0.1239.0> accepting AMQP connection <0.1239.0> (172.23.0.20:41520 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:31.823490+00:00 [error] <0.1239.0> Error on AMQP connection <0.1239.0> (172.23.0.20:41520 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:31.823490+00:00 [error] <0.1239.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:31.824410+00:00 [info] <0.1239.0> closing AMQP connection <0.1239.0> (172.23.0.20:41520 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:34.221210+00:00 [info] <0.1244.0> accepting AMQP connection <0.1244.0> (172.23.0.20:41526 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:34.224008+00:00 [error] <0.1244.0> Error on AMQP connection <0.1244.0> (172.23.0.20:41526 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:34.224008+00:00 [error] <0.1244.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:34.225188+00:00 [info] <0.1244.0> closing AMQP connection <0.1244.0> (172.23.0.20:41526 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:36.648881+00:00 [info] <0.1249.0> accepting AMQP connection <0.1249.0> (172.23.0.20:52254 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:36.652196+00:00 [error] <0.1249.0> Error on AMQP connection <0.1249.0> (172.23.0.20:52254 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:36.652196+00:00 [error] <0.1249.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:36.653938+00:00 [info] <0.1249.0> closing AMQP connection <0.1249.0> (172.23.0.20:52254 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:39.168490+00:00 [info] <0.1254.0> accepting AMQP connection <0.1254.0> (172.23.0.20:52258 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:39.170936+00:00 [error] <0.1254.0> Error on AMQP connection <0.1254.0> (172.23.0.20:52258 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:39.170936+00:00 [error] <0.1254.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:39.172295+00:00 [info] <0.1254.0> closing AMQP connection <0.1254.0> (172.23.0.20:52258 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:41.574062+00:00 [info] <0.1259.0> accepting AMQP connection <0.1259.0> (172.23.0.20:52260 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:41.577090+00:00 [error] <0.1259.0> Error on AMQP connection <0.1259.0> (172.23.0.20:52260 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:41.577090+00:00 [error] <0.1259.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:41.578261+00:00 [info] <0.1259.0> closing AMQP connection <0.1259.0> (172.23.0.20:52260 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:43.995971+00:00 [info] <0.1264.0> accepting AMQP connection <0.1264.0> (172.23.0.20:52272 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:43.998624+00:00 [error] <0.1264.0> Error on AMQP connection <0.1264.0> (172.23.0.20:52272 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:43.998624+00:00 [error] <0.1264.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:43.999966+00:00 [info] <0.1264.0> closing AMQP connection <0.1264.0> (172.23.0.20:52272 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:46.557746+00:00 [info] <0.1269.0> accepting AMQP connection <0.1269.0> (172.23.0.20:34750 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:46.560439+00:00 [error] <0.1269.0> Error on AMQP connection <0.1269.0> (172.23.0.20:34750 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:46.560439+00:00 [error] <0.1269.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:46.561943+00:00 [info] <0.1269.0> closing AMQP connection <0.1269.0> (172.23.0.20:34750 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-resource-manager-service-prod  | [15/Jul/2025 10:47:48] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:48.995659+00:00 [info] <0.1274.0> accepting AMQP connection <0.1274.0> (172.23.0.20:34754 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:48.997975+00:00 [error] <0.1274.0> Error on AMQP connection <0.1274.0> (172.23.0.20:34754 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:48.997975+00:00 [error] <0.1274.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:48.999109+00:00 [info] <0.1274.0> closing AMQP connection <0.1274.0> (172.23.0.20:34754 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:51.394882+00:00 [info] <0.1283.0> accepting AMQP connection <0.1283.0> (172.23.0.20:34762 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:51.397109+00:00 [error] <0.1283.0> Error on AMQP connection <0.1283.0> (172.23.0.20:34762 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:51.397109+00:00 [error] <0.1283.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:51.398236+00:00 [info] <0.1283.0> closing AMQP connection <0.1283.0> (172.23.0.20:34762 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-classifier-service-prod        | [15/Jul/2025 10:47:52] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-learning-service-prod          | "GET /admin/login/ HTTP/1.1" 200 4193
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:53.865178+00:00 [info] <0.1288.0> accepting AMQP connection <0.1288.0> (172.23.0.20:34770 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:53.867777+00:00 [error] <0.1288.0> Error on AMQP connection <0.1288.0> (172.23.0.20:34770 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:53.867777+00:00 [error] <0.1288.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:53.869186+00:00 [info] <0.1288.0> closing AMQP connection <0.1288.0> (172.23.0.20:34770 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-user-service-prod              | [15/Jul/2025 10:47:54] "GET /admin/login/ HTTP/1.1" 200 4178
pl-ai-data-analysis-service-prod     | [15/Jul/2025 10:47:54] "GET /admin/login/ HTTP/1.1" 200 4168
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:56.294771+00:00 [info] <0.1304.0> accepting AMQP connection <0.1304.0> (172.23.0.20:34706 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:56.297312+00:00 [error] <0.1304.0> Error on AMQP connection <0.1304.0> (172.23.0.20:34706 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:56.297312+00:00 [error] <0.1304.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:56.298588+00:00 [info] <0.1304.0> closing AMQP connection <0.1304.0> (172.23.0.20:34706 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:58.729323+00:00 [info] <0.1309.0> accepting AMQP connection <0.1309.0> (172.23.0.20:34716 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:58.731733+00:00 [error] <0.1309.0> Error on AMQP connection <0.1309.0> (172.23.0.20:34716 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:58.731733+00:00 [error] <0.1309.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:47:58.732903+00:00 [info] <0.1309.0> closing AMQP connection <0.1309.0> (172.23.0.20:34716 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:01.179528+00:00 [info] <0.1314.0> accepting AMQP connection <0.1314.0> (172.23.0.20:34724 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:01.184533+00:00 [error] <0.1314.0> Error on AMQP connection <0.1314.0> (172.23.0.20:34724 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:01.184533+00:00 [error] <0.1314.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:01.186352+00:00 [info] <0.1314.0> closing AMQP connection <0.1314.0> (172.23.0.20:34724 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:03.597889+00:00 [info] <0.1319.0> accepting AMQP connection <0.1319.0> (172.23.0.20:34738 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:03.600234+00:00 [error] <0.1319.0> Error on AMQP connection <0.1319.0> (172.23.0.20:34738 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:03.600234+00:00 [error] <0.1319.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:03.601441+00:00 [info] <0.1319.0> closing AMQP connection <0.1319.0> (172.23.0.20:34738 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:06.001031+00:00 [info] <0.1324.0> accepting AMQP connection <0.1324.0> (172.23.0.20:50974 -> 172.23.0.5:5672)
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:06.003374+00:00 [error] <0.1324.0> Error on AMQP connection <0.1324.0> (172.23.0.20:50974 -> 172.23.0.5:5672, state: starting):
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:06.003374+00:00 [error] <0.1324.0> PLAIN login refused: user 'guest' - invalid credentials
pl-ai-rabbitmq-prod                  | 2025-07-15 10:48:06.004405+00:00 [info] <0.1324.0> closing AMQP connection <0.1324.0> (172.23.0.20:50974 -> 172.23.0.5:5672)
pl-ai-rag-service-prod               | RabbitMQ non ancora pronto, aspetto 2 secondi...
